### ISSUE TITLE
Per app dark theme control

### ISSUE THREAD

**Issue by itoshkov** [donadigo/appeditor#61]
There are many non-elementary OS apps, which have nice dark themes and which look even better if the window decorations are dark as well. Would it be possible to add a checkbox to control that?

**Comment by donadigo** [comment]
Yeah I think this is pretty possible, but before implementing this I would like to make sure that the elementary review team will accept this change and the user gets notified about possible inconsistencies in the app when switching to dark theme.

Also, I'm not really sure that it'll work for anything else besides Gtk3 apps.

**Comment by itoshkov** [comment]
I meant that it would switch just the window manager decorations. The app can be switched manually to the dark theme.

**Comment by biswaz** [comment]
How can I do that manually? Is there a command line argument for turning on dark mode for an app?

**Comment by donadigo** [comment]
On command line: `GTK_THEME=elementary:dark app`, but note that this won't work perfectly with every app. Right now this won't work in AppEditor since it doesn't environmental variables yet.

**Comment by itoshkov** [comment]
> On command line: `GTK_THEME=elementary:dark app`, but note that this won't work perfectly with every app. Right now this won't work in AppEditor since it doesn't environmental variables yet.

What if we use `env GTK_THEME=elementary:dark app` as command in the AppEditor? Souldn't this work?

**Comment by donadigo** [comment]
It is possible that this would work, yes.

**Comment by biswaz** [comment]
It works. Tested on the libre suite.
